### ISSUE TITLE
Cannot find file: Does not match the corresponding name on disk.

### ISSUE THREAD

**Issue by timfrankentwf** [wintergatan-community/virtual-mmx#32]
I tried to run the program, but got the following error:
Branch: dev
```js
Cannot find file: 'ProgrammingWheel.tsx' does not match the corresponding name on disk: '.\src\components\programEditor\ProgramEditor'
```
Renaming folder **ProgramEditor** to **programEditor** in `virtual-mmx\src\components\` solved the issue.

**Comment by mozi-h** [comment]
I'm not seeing the mentioned file in https://github.com/wintergatan-community/virtual-mmx/tree/dev/src/components

**Comment by timfrankentwf** [comment]
I see that the issue has been resolved as result of the following commit `f25cf7d` by @Zakgriffin about 4 days ago.
Within this commit the folder has been renamed from **ProgramEditor** to **programmingWheel**.
